content/en/docs/zero-code/java/quarkus/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Quarkus
```

Comment:
Perhaps to not confuse with the OTel agent instrumentation
```suggestion
title: Quarkus native instrumentation
```

Reply:
The native here is ambiguous. Native means instrumentation for Quarkus or Native mode instrumentation for Quarkus, which is misleading. 
Will set "Quarkus instrumentation"